src/main/java/org/sonarqube/gradle/SonarUtils.java

```diff
@@ -38,6 +40,11 @@
 
 public class SonarUtils {
 
+  private static final Pattern COVERAGE_REPORT_PATH_PROPERTY_PATTERN = Pattern.compile(
```

Comment:
In the scanner for maven we use the pattern: `"^sonar\\..*[rR]eportPaths?$"`. Should we try to be consistent?
Does the pattern need to be necessarily  `CASE_INSENSITIVE`?

Reply:
I intentionally made the pattern more complex to cover both external reports and [coverage reports](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/test-coverage/test-coverage-parameters/). WDYT?

Reply:
The case insensitive part is not absolutely necessary but I felt like it meant the pattern more readable. WDYT?

Reply:
Actually, this is a good idea and it would be nice to have it in the Scanner for Maven as well :)
Do you think that having 2 regexes would be more readable, or is it clear enough by having a more attentive look?